.gitmodules

```diff
@@ -16,9 +16,12 @@
 [submodule "telemetry"]
 	path = telemetry
 	url = https://github.com/Azure/device-telemetry-java
-[submodule "telemetry-agent"]
-	path = telemetry-agent
-	url = https://github.com/Azure/telemetry-agent-java
 [submodule "config"]
 	path = config
 	url = https://github.com/Azure/pcs-config-java.git
+[submodule "asa-manager"]
+	path = asa-manager
+	url = https://github.com/Azure/asa-manager-dotnet
+[submodule "auth"]
```

Comment:
for the future, we should rename this to "user-management", also align all the modules to the names used here https://user-images.githubusercontent.com/3317135/31914374-44a4be80-b7ff-11e7-86b2-19845ab65d7a.png